contracts/staking/RewardDelegators.sol

```diff
@@ -245,6 +259,26 @@ contract RewardDelegators is
             );
 
             _aggregateReward = _aggregateReward + _reward;
+
+            // assuming this is pond
+            if(_tokenId == pondToken){
+                totalDelegations += clusters[_cluster].totalDelegations[_tokenId];
```

Comment:
instead just add pond and mpond token delegations outside the for loop as this for might not always have pond and mpond in `_tokens` array.